definitions/npm/node-fetch_v2.x.x/flow_v0.44.x-/node-fetch_v2.x.x.js

```diff
@@ -0,0 +1,153 @@
+import type { Agent } from "http";
+import type { URLSearchParams } from "url";
```

Comment:
Those imports needs to be moved inside the module declaration, otherwise they will resolve to `any`.

Reply:
I decided to switch to TypeScript, so if you want you can correct my PR or reject it.